fuzz/targets/x509_unsigned.rs

```diff
@@ -0,0 +1,54 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+//! Fuzz tests for the X.509 certificate parser, including signature
+//! validation.
+
+#![no_main]
+
+use libfuzzer_sys::fuzz_target;
+
+use manticore::cert;
+use manticore::cert::Algo;
+use manticore::cert::Cert;
+use manticore::cert::CertFormat;
+use manticore::cert::PublicKeyParams;
+use manticore::crypto::sig;
+
+/// A `Ciphers` that blindly accepts all signatures.
+struct NoVerify;
```

Comment:
Can you not reused the existing `NoVerify`?

Reply:
That would require exposing it as part of Manticore's API. If this starts happening enough I may consider putting it somewhere global, but definitely not in manticore.